docs/mllib-linear-methods.md

```diff
@@ -282,8 +277,8 @@ public class SVMClassifier {
     System.out.println("Area under ROC = " + auROC);
 
     // Save and load model
-    model.save(sc.sc(), "myModelPath");
-    SVMModel sameModel = SVMModel.load(sc.sc(), "myModelPath");
+    model.save(sc, "myModelPath");
```

Comment:
Hi xiangrui, the sc here seems to be an instance of SparkContext (refer to line 251). And the original code has compile error.